release/e2e.py

```diff
@@ -1317,11 +1380,35 @@ def _run(logger):
                 _process_finished_command(
                     session_controller=session_controller, scd_id=scd_id)
             else:
+                timeout_type = ""
+                runtime = None
+                if isinstance(e, CommandTimeoutError):
+                    timeout_type = "timeout"
+                    runtime = 0
+                elif (isinstance(e, PrepareCommandTimeoutError)
+                      or isinstance(e, FileSyncTimeoutError)
+                      or isinstance(e, SessionTimeoutError)):
+                    timeout_type = "infra_timeout"
+                    runtime = None
+                elif isinstance(e, RuntimeError):
+                    timeout_type = "runtime_error"
+                    runtime = 0
+                else:
+                    timeout_type = "unknown timeout"
+                    runtime = None
```

Comment:
How about 0 for this case?

Reply:
Theoretically this shouldn't be reachable, but I just added just in case not to crash the nightly dashboard. I think this means there's an exception that we don't catch (which means that's the releaser problem -> infra failures)

Reply:
Lmk if you still think 0 is better (in that case, I will fix that in a follow up PR)

Reply:
If it's an unknown exception I think we should err on over-reporting (kind of like a check failure). Otherwise, we might be silently ignoring app-level failures.